packages/block-library/src/image/editor.scss

```diff
@@ -124,3 +124,14 @@
 		caption-side: bottom;
 	}
 }
+
+.block-editor-block-list__block[data-type="core/image"] {
+	.wp-block-image-replace .components-placeholder__label {
```

Comment:
I think it'd be preferable to define these styles with the media-placeholder component, as there's a good chance we'll also want to bring this flow to other media blocks (video, audio, media & text, gallery etc.)

Using a separate element for the preview image should mean we don't have to do anything clever to change the way the label appears.